### PR TITLE
1.5.0 Release

### PR DESCRIPTION
**Pull request by shiftkey**
## Overview

Adding the release notes for the 1.5.0 release.

If there are any additional changes that need to be made available for 1.5.0, please open a PR targeting `1-5-0-release-branch`. **Do not push non-release-notes changes directly to this branch.**

## Description

- [x] approve release notes
- [x] publish to `production` (Wedesday 8am PST) 
- [x] publish `1.5.1-beta0` to `beta` (once production is shipped and smoke tested)

## Release notes

Notes: no-notes

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,31 @@
 {
   "releases": {
+    "1.5.0": [
+      "[New] Repository switcher has a convenient \"Add\" button to add other repositories - #5878",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[New] Clone, create, or add repositories right from the repository dropdown - #5878",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+      "[New] macOS tray icon now supports drag-and-drop to add local repositories - #5048",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[New] Drag-and-drop to add local repositories from macOS tray icon - #5048",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+      "[Fixed] Diff viewer blocks keyboard navigation using reverse tab order - #2794",
+      "[Fixed] Launching Desktop from browser always asks to clone repository - #5913",
+      "[Fixed] Publish dialog displayed on push when repository is already published - #5936",
+      "[Improved] \"Publish Repository\" dialog handles emoji characters - #5980. Thanks @WaleedAshraf!",
+      "[Improved] Avoid repository checks when no path is specified in \"Create Repository\" dialog - #5828. Thanks @JakeHL!",
+      "[Improved] Clarify the direction of merging branches - #5930. Thanks @JQuinnie!",
+      "[Improved] Default commit summary more explanatory and consistent with GitHub.com - #6017. Thanks @Daniel-McCarthy!",
+      "[Improved] Display a more informative message on merge dialog when branch is up to date - #5890",
+      "[Improved] Getting a repository's status only blocks other operations when absolutely necessary - #5952",
+      "[Improved] Merge dialog displays current branch in header - #6027",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Display current branch in header of merge dialog - #6027",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+      "[Improved] Sanitize repository name before publishing to GitHub - #3090. Thanks @Daniel-McCarthy!",
+      "[Improved] Show the branch name in \"Update From Default Branch\" menu item - #3018. Thanks @a-golovanov!",
+      "[Improved] Updated license and .gitignore templates for initializing a new repository - #6024. Thanks @say25!"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Update license and .gitignore templates for initializing a new repository - #6024. Thanks @say25!"
```

File: app/package.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "productName": "GitHub Desktop",
   "bundleID": "com.github.GitHubClient",
   "companyName": "GitHub, Inc.",
-  "version": "1.5.0-beta4",
+  "version": "1.5.0-beta5",
```
Comment (billygriffin) [inline app/package.json]: This is probably just my own ignorance, but just checking whether this is correct or whether it should be 1.5.0 (without the beta) for the prod release.

Comment (shiftkey) [reply app/package.json]: This is correct, and Central would complain loudly without a new version to use for the release. I'll push a commit to this branch to make that clear.

**Comment by say25** [comment]
@desktop/engineering just trying to learn process, would a push to beta be part of this?

If so do we push said beta off `master` or off this branch?

**Comment by shiftkey** [comment]
> (I think the idea is to have the first beta following a production release be as close to that production release as possible.)

This is correct. After the production release is published these will be the two versions in the wild:

 - production: `1.5.0`
 - beta: `1.5.0-beta5`

We need to bump the beta version to ensure it's a version greater than what's in production, and also ensure that any hotfixes targeting the release branch (like #6153) are also pushed to beta users.

Then we figure out what we're going to pull into `1.5.1` and start planning the `1.5.1-beta1` release.

**Pull request merged**